modules/standard/UtilReplicatedVar.chpl

```diff
@@ -59,38 +63,60 @@ How to use replicated variables:
     // access directly a remote copy on the locale 'remoteLoc' (read or write)
     ... rcRemote(myRepVar, remoteLoc) ...
 
-Advanced: replicate over a subset of locales 'myLocales'.
-// 'myLocales' must be "consistent", as defined for ReplicatedDist.
-Use it as above, except modify the variable declarations as follows:
+For more examples, see :proc:`rcExample` and :proc:`rcExampleOverLocales`
+in the module source code.
+
+----------------------------------------------
+Advanced: replicating over a subset of locales
+----------------------------------------------
+
+To replicate a variable over a subset of locales, say ``myLocales``,
+modify the above variable declarations as follows::
 
     var myRepVar: [rcDomainBase dmapped ReplicatedDist(myLocales,
                      "over which to replicate 'myRepVar'")] MyType;
     var collected: [myLocales.domain] MyType;
 
-See also rcExample() and rcExampleOverLocales() in this file.
-*/
+``myLocales`` must be "consistent", as defined for ReplicatedDist.
+That is, for each ``ix`` in ``myLocales.domain``,
+``myLocales[ix]`` must equal ``Locales[ix]``.
 
-/* */
+Tip: if the domain of the desired array of locales cannot be described
+as a rectangular domain (which could be strided, multi-dimensional,
+and/or sparse), make that array's domain associative over int.
+*/
```

Comment:
Since you have generated a section header and one has to scroll before reaching code defined in the module, I would recommend ending the module comment with a descriptive header, so that documentation readers can easily skip to the function definitions, etc.